ChatSDK/ChatSDKFirebaseAdapter/Classes/Firebase/CCMessageWrapper.m

```diff
@@ -99,7 +99,8 @@ -(NSDictionary *) serialize {
              b_Type: _model.type,
              b_Date: [FIRServerValue timestamp],
              b_UserFirebaseID: _model.userModel.entityID,
-             b_ReadPath: self.initialReadReceipts};
+             b_ReadPath: self.initialReadReceipts,
+             b_Meta: _model.metaDictionary};
```

Comment:
What happens if the meta is Nil?

Reply:
Meta should never be nil.

Currently we are only dealing with mention meta data, it might be that in the future we add image message meta etc. For functions with meta data we set .metaDictionary as the meta. For all other message types we just set the meta Dictionary as a nil dictionary .metaDictionary = @{}. This lays the foundation for meta data without requiring us to add checks. 

Note: I forgot to add this in this pull request - in further testing I found this to be an issue so have added empty meta dictionaries to the other message types